en/micro-integrator/docs/references/connectors/microsoft-dynamics365-connector/microsoft-dynamics365-configuration.md

```diff
@@ -0,0 +1,157 @@
+       
+    4. Click **All admin centers** and click **Dynamics 365** icon. It will navigate to the **Power Platform admin center**
+
+       <img src="/assets/img/connectors/MSdynamics365-all-admin-center.png" title="Dynamics365 all admin center" width="800" alt="Dynamics365 all admin center"/>
+
+    5. Select created environment and click **Open environment**.
+    
+       <img src="/assets/img/connectors/MSdynamics365-select-environment.png" title="Dynamics365 select environment" width="800" alt="Dynamics365 select environment"/>
+       
+    6. Navigate to Settings -> Security -> Users.
+    
+       <img src="/assets/img/connectors/MSdynamics365-settings.png" title="Dynamics365 settings" width="800" alt="Dynamics365 settings"/>
+        
+       <img src="/assets/img/connectors/MSdynamics365-Users.png" title="Dynamics365 users" width="800" alt="Dynamics365 users"/>
+    
+    7. Choose **Application Users** in the view filter.
+    
+       <img src="/assets/img/connectors/MSdynamics365-filter-application-user.png" title="Filter application users" width="800" alt="Filter application users"/>
+    
+    8. Select -> **New**.
+       
+    9. In the Application User form, enter the required information.
+    
+       <img src="/assets/img/connectors/MSdynamics-new-application-user.png" title="Dynamics365 new application user" width="800" alt="Dynamics365 new application user"/>
+
+       The user name information must not match a user that exists in the Azure Active Directory.
+       In the **Application ID** field, enter the application ID of the app you registered earlier in the Azure AD.
+       
+    10. If all goes well, after selecting SAVE, the Application ID URI and Azure AD Object Id fields will auto-populate with correct values. 
+    
+       <img src="/assets/img/connectors/MSdynamics365-created-application-user.png" title="Dynamics365 created application user" width="800" alt="Dynamics365 created application user"/> 
+    
+    11. Before exiting the user form, choose MANAGE ROLES and assign a security role to this application user so that the application user can access the desired organization data.
+   
+## Generate Access Token 
+
+After setup Azure and Microsoft Dynamics 365 CRM configurations you can get access token by invoking the following HTTP request.
+
+POST URL:
+
+https://login.microsoftonline.com/<tenant id> /oauth2/token
+
+Header:Content-Type: application/json
+
+Body: x-www-form-urlencoded
+
+
+| Key        | Value |
+| ------------- |-------------|
+| client_id    | Application ID of the registered app in Azure. |
+| resource      |https://trial.crm.dynamics.com (Dynamics 365 Online Insance URL |
+| Client_secret |Key value from the registered app in Azure|
+| Grant_type    |client_credentials|
+
+Making a Request using Postman
+
+<img src="/assets/img/connectors/MSdynamics365-access-token.png" title="Obtaining access token" width="800" alt="Obtaining access token"/>
```

Comment:
Need to mention that you want to  save which fields to proceed with configuring the connector.

Reply:
Added.